resources/candlepin-lint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -xe
+
+env | sort
+echo
+
+docker pull docker-registry.usersys.redhat.com/candlepin/candlepin-postgresql || true
+echo "Docker pull completed"
+
+# The docker container test script will know to copy out
+echo "Using workspace: $WORKSPACE"
+mkdir -p $WORKSPACE/artifacts/
+
+# make selinux happy via http://stackoverflow.com/a/24334000
+chcon -Rt svirt_sandbox_file_t $WORKSPACE//artifacts/
+
+# Fix PR author if necessary
+if [ "$ghprbPullAuthorEmail" == "" ]; then
+  `git log -n 1 --format="export ghprbPullAuthorEmail=%aE"`
+fi
+
+echo "Using PR author e-mail: $ghprbPullAuthorEmail"
```

Comment:
This PR author bit was part of a change I attempted to add a while ago where we would have the PR checks e-mail the submitter directly when an error occurred rather than relying on global e-mail spam or IRC messages. I eventually got pulled to other tasks before ever getting this functioning properly (hence why it's only ever used for this echo statement in the current job system) -- something with the way a job is invoked changing the availability or correctness with the PR author fetching bits.

With the new way of creating these jobs, do we have any ability to fix/correct various plugins or otherwise get the PR author in a consistent way? If so, this block should be dropped and we should migrate toward only contacting the PR author in the event of a failure.